documentation/quarkus.asciidoc

```diff
@@ -11,17 +11,17 @@ Additional things like extensions will be available on the  https://github.com/d
+* If you are new to devon4j, also take a look at devon4j's recommendations on general best practices. Check out the chapters on link:architecture.asciidoc[architecture design], link:guide-structure-modern.asciidoc[project structuring] and link:coding-conventions.asciidoc[coding conventions]. Follow the referenced links to explore a topic in more depth.
 
 * If you are an experienced Spring developer and want to get in touch with Quarkus, read our link:quarkus/getting-started-for-spring-developers.asciidoc[Getting started with Quarkus for Spring developers] guide.
 
-* If you're looking to build your first Quarkus application, the Quarkus website offers some good link:https://quarkus.io/get-started/[getting started guides]. Also check out our link:quarkus/quarkus-template.asciidoc[Quarkus template guide], which gives you some recommendations on extensions and frameworks to use. It also provides some links to the link:https://code.quarkus.io/[Quarkus code generator] with preselected configurations you can use to create your application.
+* If you're looking to build your first Quarkus application, the Quarkus website offers some good https://quarkus.io/get-started/[getting started guides]. Also check out our link:quarkus/quarkus-template.asciidoc[Quarkus template guide], which gives you some recommendations on extensions and frameworks to use. It also provides some links to the https://code.quarkus.io/[Quarkus code generator] with preselected configurations you can use to create your application.
 
-* If you want to have a Quarkus sample application using devon4j recommendations, check out our link:https://github.com/devonfw-sample/devon4quarkus-reference[Quarkus reference application].
+* If you want to have a Quarkus sample application using devon4j recommendations, check out our https://github.com/devonfw-sample/devon4quarkus-reference[Quarkus reference application].
 
-* If you already have some experience with devon4j and Quarkus and need more information on a specific topic, check out our link:https://devonfw.com/website/pages/docs/devon4j.wiki_devon4j.asciidoc_guides_2.html[Quarkus guides]. If you don't find what you are looking for there, check out the link:https://devonfw.com/website/pages/docs/devon4j.asciidoc_general.html[general section]. devon4j uses general solutions for Java, so solutions for both Quarkus and Spring are documented there.
+* If you already have some experience with devon4j and Quarkus and need more information on a specific topic, check out our https://devonfw.com/website/pages/docs/devon4j.wiki_devon4j.asciidoc_guides_2.html[Quarkus guides]. If you don't find what you are looking for there, check out the https://devonfw.com/website/pages/docs/devon4j.asciidoc_general.html[general section]. devon4j uses general solutions for Java, so solutions for both Quarkus and Spring are documented there.
```

Comment:
@GuentherJulian could you please provide the actual asciidoc internal links instead of the devonfw.com link all over the references?

Reply:
I discussed with @SchettlerKoehler and it seems to be really a problem of the current document generation. So we'll keep it like that for the moment for the guides section.
We also discussed to potentially setup a campus team next year for antora in-depth evaluation and migration to get rid of such issues in future as well as to get rid of the website complexity as it is now for the docs section, which is coming with a lot of pitfalls due to custom scripting and chained workarounds.